plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlClient.java

```diff
@@ -900,7 +900,7 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
             createTable(session, tableMetadata, tableMetadata.getTable().getTableName());
         }
         catch (SQLException e) {
-            boolean exists = SQL_STATE_ER_TABLE_EXISTS_ERROR.equals(e.getSQLState());
+            boolean exists = ER_TABLE_EXISTS_ERROR == e.getErrorCode();
```

Comment:
We can avoid a yoda condition now because `e.getErrorCode()` is primitive unlike `e.getSQLState()`.